pages/edit_news_page.py

```
@@ -63,3 +67,20 @@ def click_edit_news_btn(self):
     def click_yes_cancel_btn(self):
         cancel_btn = self.get_wait().until(EC.element_to_be_clickable(self.YES_CANCEL_BTN))
         cancel_btn.click()
+
+    @allure.step("Add news comment")
+    def add_news_comment(self):
+        enter_content = self.get_wait().until(EC.element_to_be_clickable(self.COMMENT_FIELD))
+        assert enter_content.is_enabled()
```

Ruff (0.12.2)

Use of `assert` detected

(S101)